using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -294,7 +348,11 @@ If the provided URL is nil or empty, it is returned as is. Otherwise, the follow
 + (void) appendToUrl: (nullable NSURL*) baseUrl withCallback: (nullable void (^) (NSURL* __nullable urlWithVisitorData)) callback;
 ```
 
-#### **Examples**
+*o baseUrl* is the URL to which the visitor info needs to be appended. Returned as is if it is nil or empty.
```

Comment:
- Move the parameters to after the code example.
- Reword the 2nd sentence to read: "If the visitor information is nil or empty, the URL is returned as is." and move to the next line.